acceptance_tests/features/fixtures.py

```diff
@@ -162,6 +162,10 @@ def _setup_data_with_internal_user_and_collection_exercise_to_specific_status(co
     response = create_test_business_collection_exercise(survey_id, period, short_name, ce_name,
                                                         survey_type, stop_at_state=stop_at_state)
 
+    for key, date in response['dates'].items():
```

Comment:
Rather than doing this, wouldn't it be better to set the TZ info in `generate_collection_exercise_dates_from_period`?

Reply:
Is this still needed after the changes @ryangrundy7